onnxruntime/contrib_ops/cuda/math/fft_ops.cc

```diff
@@ -19,7 +19,7 @@ void SetFFTState(FFTState* state,
                  cudaDataType exec_type) {
   memset(state, 0, sizeof(FFTState));
   state->signal_ndim = signal_ndim;
-  for (int32_t i = 0; i < signal_dims.size(); ++i) {
+  for (auto i = 0; i < static_cast<int64_t>(signal_dims.size()); ++i) {
```

Comment:
Don't use auto. Please explicitly put a type there. 99% C/C++ programmers  do not know how the types of integer literals are determined. The rule is very complicated. We should use only auto when the type is obvious.